mungegithub/webhooks-publisher/README.md

```diff
@@ -14,3 +17,32 @@ kubectl create configmap webhook-config-v10 --from-file=config.yaml
 
 kubectl apply -f deployment.yaml
 ```
+
+How to use
+----------
+
+`config.yaml` contains the configuration:
+
+- `project` is the project that has the PubSub queue
+- Each item in `paths` is the path it listens to webhooks and maps to the Github
+  `secret` and the PubSub `topic` where it should publish
+
+If you want to listen to a repository:
```

Comment:
How do we make Github send us the webhook notifications ? Is that documented somewhere too ?